domains/davechbga.json

```diff
@@ -4,6 +4,6 @@
         "email": "davechbga@gmail.com"
     },
     "record": {
-        "CNAME": "davechbga.vercel.app"
+        "TXT": "_vercel.davechbga.is-a.dev"
```

Comment:
TXT verification records should be under another file. See docs: https://docs.is-a.dev/guides/vercel